MaterialDesignThemes.Wpf/PasswordBoxAssist.cs

```diff
@@ -1,8 +1,10 @@
-﻿using System.Windows.Data;
+using System.Windows.Data;
+using System.Windows.Documents;
+using Microsoft.Xaml.Behaviors;
 
 namespace MaterialDesignThemes.Wpf;
 
-public static class PasswordBoxAssist
+public class PasswordBoxAssist : Behavior<PasswordBox>
```

Comment:
I think it might be better to move the behavior to its own class so there is no confusion between what is a behavior and what is an "Assist" (attached property) class. Perhaps named something like "PasswordBoxBehavior"?

Reply:
I agree. It wasn't easy to split out with the first behavior code I wrote, but with what I ended up with it should be easy to extract to a different class.